domain/application/state/charm.go

```diff
@@ -200,9 +200,9 @@ func (s *State) IsCharmAvailable(ctx context.Context, id corecharm.ID) (bool, er
 	ident := charmID{UUID: id.String()}
 
 	query := `
-SELECT charm.available AS &charmAvailable.available
+SELECT &charmAvailable.*
```

Comment:
Not that important, but for these queries I prefer `SELECT &charmAvailale.available` because it makes it clear that thats all you're selecting there.

Reply:
Simon and I prefer this